source/crashHandler.py

```diff
@@ -0,0 +1,182 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
can you move this to be a file at `source\utils\_crashHandler.py`. I've realised this should probably not be part of the public API, and works better in `utils`